screens/UpcomingScreen.js

```diff
@@ -84,14 +84,18 @@ class UpcomingScreen extends React.Component {
           />
         }>
 
-        <Text style={styles.chefText}> Events as Host </Text>
-
-        {this.props.currentChef ? (
```

Comment:
We still want this header section here conditionally, based on if the user is registered as a chef.